tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -84,6 +84,13 @@ Given('the setting {string} of app {string} has been set to {string}', function
     ])
 })
 
+Given('app {string} has been enabled', function (app) {
+  return occHelper.runOcc(
```

Comment:
It'd be better to do a `fetch` request, rather than `runOcc`.

You can do a following curl example:
```
curl -u admin:admin http://localhost/core/ocs/v1.php/cloud/apps/{app_name} -X POST
# eg: curl -u admin:admin http://localhost/core/ocs/v1.php/cloud/apps/federation -X POST
```

Reply:
this apps endpoint is from a time where we did not have an occ endpoint. If we use more the occ endpoint, we can depreciate endpoints that only do what occ can do

Reply:
@individual-it, I didn't understand the comment. Are you suggesting to use occ endpoints or to use OCS endpoints?

Reply:
@skshetry  I was talking about the testing app endpoint, but you are right, if there is an official API endpoint, lets use that